### PR TITLE
Added the new 'influx pkg stack' commands

### LINKED ISSUE #876: Document 'influx pkg stack' command(s)
PR: https://github.com/influxdata/influxdb/pull/17448

### PR DESCRIPTION
**Pull request by sanderson**
Closes #876 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/reference/cli/influx/pkg/stack/init.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Usage
+```
+influx pkg stack init [flags]
+```
+
+## Flags
+| Flag                        | Description                     | Input type      | {{< cli/mapped >}} |
+|:----                        |:-----------                     |:----------:     |:------------------ |
+| `-h`, `--help`              | Help for the `init` command     |                 |                    |
+| `--json`                    | Output data as JSON             |                 |                    |
```
Comment (jsteenb2) [inline content/v2.0/reference/cli/influx/pkg/stack/init.md]: ahh shooot, glad I saw this, I forgot to normalize the json flag here on this command with all the others that are coming up. The apply and summary commands were also updated to use json. The flag looks like the following:

```
   --json                       Output data as json; defaults false; Maps to env var $INFLUX_OUTPUT_JSON
```

apply flags updated here:
https://github.com/influxdata/influxdb/blob/master/cmd/influx/pkg.go#L100

summary flags updted here:
https://github.com/influxdata/influxdb/blob/master/cmd/influx/pkg.go#L326

this func will be used in the future to normalize all json/hide headers type of flags:

https://github.com/influxdata/influxdb/blob/master/cmd/influx/main.go#L444

File: content/v2.0/reference/cli/influx/pkg/_index.md [hunk 2]
```diff
@@ -17,12 +17,13 @@ influx pkg [flags]
...
-|:-------                                              |:-----------                                                       |
-| [export](/v2.0/reference/cli/influx/pkg/export/)     | Export existing resources as a template                           |
-| [summary](/v2.0/reference/cli/influx/pkg/summary/)   | Summarize the provided template                                   |
-| [validate](/v2.0/reference/cli/influx/pkg/validate/) | Validate the provided template                                    |
+## Subcommands
+| Subcommand                                           | Description                             |
+|:-------                                              |:-----------                             |
+| [export](/v2.0/reference/cli/influx/pkg/export/)     | Export existing resources as a template |
+| [stack](/v2.0/reference/cli/influx/pkg/stack/)       | Manage stacks                           |
+| [summary](/v2.0/reference/cli/influx/pkg/summary/)   | Summarize the provided template         |
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/pkg/_index.md]: specified template?

File: content/v2.0/reference/cli/influx/pkg/_index.md [hunk 3]
```diff
@@ -17,12 +17,13 @@ influx pkg [flags]
...
+| [validate](/v2.0/reference/cli/influx/pkg/validate/) | Validate the provided template          |
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/pkg/_index.md]: specified template?

File: content/v2.0/reference/cli/influx/pkg/_index.md [hunk 4]
```diff
@@ -17,12 +17,13 @@ influx pkg [flags]
...
-## Available commands
-| Command                                              | Description                                                       |
-|:-------                                              |:-----------                                                       |
-| [export](/v2.0/reference/cli/influx/pkg/export/)     | Export existing resources as a template                           |
-| [summary](/v2.0/reference/cli/influx/pkg/summary/)   | Summarize the provided template                                   |
-| [validate](/v2.0/reference/cli/influx/pkg/validate/) | Validate the provided template                                    |
+## Subcommands
+| Subcommand                                           | Description                             |
+|:-------                                              |:-----------                             |
+| [export](/v2.0/reference/cli/influx/pkg/export/)     | Export existing resources as a template |
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/pkg/_index.md]: Export resources as a template

Comment (kelseiv) [inline content/v2.0/reference/cli/influx/pkg/_index.md]: maybe link to definition of resources

File: content/v2.0/reference/cli/influx/pkg/stack/_index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+description: The 'influx pkg stack' command and its subcommands manage InfluxDB stacks.
+menu:
+  v2_0_ref:
+    name: influx pkg stack
+    parent: influx pkg
+weight: 101
+v2.0/tags: [templates]
+---
+
+The `influx pkg stack` command manages InfluxDB stacks.
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/pkg/stack/_index.md]: maybe define stacks within context of templates?

File: content/v2.0/reference/cli/influx/pkg/stack/init.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+---
+title: influx pkg stack init
+description: The 'influx pkg stack init' command initializes and InfluxDB stack.
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/pkg/stack/init.md]: an InfluxDB stack

File: content/v2.0/reference/cli/influx/pkg/stack/init.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+menu:
+  v2_0_ref:
+    name: influx pkg stack init
+    parent: influx pkg stack
+weight: 201
+v2.0/tags: [templates]
+---
+
+The `influx pkg stack init` command initializes and InfluxDB stack.
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/pkg/stack/init.md]: an

File: content/v2.0/reference/cli/influx/pkg/stack/_index.md (hunk 5 above)
Comment (sanderson) [reply content/v2.0/reference/cli/influx/pkg/stack/_index.md]: I'm going to save this for the task based version of this doc. Once that exists, I'll add it as a related link to this doc. This update is laying down the foundation for stacks and they're still likely to change a lot. My goal here is to just document the CLI. We can cover the conceptual stuff later.

**Pull request merged**